flycheck-coala.el

```diff
@@ -34,10 +34,10 @@
 (flycheck-define-checker coala
   "A checker using coala.
 See URL `https://coala.io'."
-  :command ("coala" "--format" "{line}:{column}:{severity_str}:{message}" "--find-config" "--files" source)
-  :error-patterns  ((error (or "None" line) ":" (or "None" column) ":MAJOR:" (message))
-                    (warning (or "None" line) ":" (or "None" column) ":NORMAL:" (message))
-                    (info (or "None" line) ":" (or "None" column) ":INFO:" (message)))
+  :command ("coala" "--format" "{line}:{column}:{severity_str}:{message}" "--find-config" "--limit-files" source-original)
+  :error-patterns  ((error line-start (or "None" line) ":" (or "None" column) ":MAJOR:" (message) line-end)
+                    (warning line-start (or "None" line) ":" (or "None" column) ":NORMAL:" (message) line-end)
+                    (info line-start (or "None" line) ":" (or "None" column) ":INFO:" (message) line-end))
```

Comment:
Don't you have to adapt the coala `--format` to also emit line-starts and line-ends?

Reply:
See `:error-patterns` documentation and example in http://www.flycheck.org/en/latest/developer/developing.html#writing-the-checker. The `line-start` and `line-end` are `rx` pattern elements and do not map to checker command results.